src/navs/index.js

```diff
@@ -2706,6 +2706,12 @@ export const docsMenu = {
                     icon: 'IconHogQL',
                     color: 'purple',
                 },
+                {
+                    name: 'Filtering, Visualizing, and Joining Data',
```

Comment:
```suggestion
                    name: 'Filtering, visualizing, and joining data',
```

Reply:
I'm not sure this is the right title for the sidebar, but fine for now

Reply:
same, i just wrote down whatever came to mind immediately here